lib/colorado_lottery.rb

```diff
@@ -0,0 +1,33 @@
+class ColoradoLottery
+  attr_reader :registered_contestants, :winners, :current_contestants
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def interested_and_18?(name, game)
```

Comment:
This works, but when we have a boolean method, we never need control flow. The first two conditions could be joined with an `&&`:
`name.age >= 18 && name.game_interests.include?(game.name)`

`name` should also be called contestant here to accurately represent the object.